concepts/src/concept_csv_export.py

```diff
@@ -266,29 +289,39 @@ def locale_join_snippet(name_types: list, locale: str):
 
         return "\n    ".join(snippets)
 
-    def type_name_transform(mapping_type: str):
-        return mapping_type.replace("-", "_").replace(" ", "_")
+    def name_transform(mapping_type: str):
+        return mapping_type.replace("-", "_").replace(" ", "_").replace(".", "_").replace("|", "_")
 
-    def mapping_select_snippet(mapping_type: str):
```

Comment:
This stuff adds support for separate columns per concept source, as well as support for "Name" and "Number" for the PIH sources

Reply:
Yeah still not sure why you would make this script more complicated rather than just handling these transformations downstream